plugins/generator-1.20.1/forge-1.20.1/procedures/team_get_entity_team.java.ftl

```diff
@@ -1,2 +1,2 @@
-(${input$entity} instanceof LivingEntity _teamEnt && _teamEnt.level().getScoreboard().getPlayersTeam(_teamEnt.getStringUUID()) != null ?
-	_teamEnt.level().getScoreboard().getPlayersTeam(_teamEnt instanceof Player _pl ? _pl.getGameProfile().getName() : _teamEnt.getStringUUID()).getName() : "")
\ No newline at end of file
+(${input$entity} instanceof Player _pl && _pl.level().getScoreboard().getPlayersTeam(_pl.getGameProfile().getName()) != null ?

```

Comment:
Checking #4388, the correct fix would likely be to use

`_teamEnt instanceof Player _pl ? _pl.getGameProfile().getName() : _teamEnt.getStringUUID()`-alike in both cases,

not just `_pl.getGameProfile().getName()` or this fix will land another issue like https://github.com/MCreator/MCreator/issues/4388

Reply:
When I made the blocks, I remember to have seen player's name in the Vanilla code

Reply:
Yes, but this code won't work with non-players now while it can work with non-players if implemented correctly

Reply:
Oh sorry, I didn't see correctly the code you wrote (I was just waking up 😅). I only read the getStringUUID. This makes more sense now.